inventory/byo/hosts.origin.example

```diff
@@ -113,6 +113,9 @@ openshift_release=v3.6
 # Downgrades are not supported and will error out. Be careful when upgrading docker from < 1.10 to > 1.10.
 # docker_version="1.12.1"
 
+# Specify whether to --enable-selinux on Docker (set false if using overlay/2 storage drivers before RHEL 7.4)
```

Comment:
There's one task in one role that uses it. And as far as I can see, nothing that specifies what storage driver you're using (when I configure it in `/etc/sysconfig/docker-storage` it doesn't get reconfigured). I don't know of any other reason why you would set this to false, and I suspect no one else does either because it didn't even work.